Resources/Private/Fusion/Shame.fusion

```diff
@@ -0,0 +1,3 @@
+##
+# Quick & Dirty fixes, that can be improved later on
+#
```

Comment:
I see, but didn't find it self explanatory even with the comment and is it really a common convention you want to encourage? At least I'd like to challenge that and make such a decision by gathering more consensus and reasoning.

What would happen if it wasn't a convention, and what will happen if it becomes a convention?

btw. it's mentioned at https://youtu.be/8bZikINA2UA?t=17m57s for those curious